### PR TITLE
Basic data structure

### PR DESCRIPTION
**Pull request by yuejunfeng0909**
#17 contains Item class
#21 contains ItemContainer class which works like a shelf

### CODE REVIEW

**Comment by haoyusimon** [comment]
Nice work with clear structure. However, could you add JavaDoc or leave some comments to explain what different classes such as ItemContainer/Item/ContainerList mean?

**Comment by t-l-xin** [comment]
Agreed, the Javadocs need more description

File: src/main/java/seedu/duke/model/ContainerList.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package seedu.duke.model;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a collection of ItemContainers.
+ */
+public class ContainerList {
```
Comment (haoyusimon) [inline src/main/java/seedu/duke/model/ContainerList.java:8]: This class can be used for later versions

File: src/main/java/seedu/duke/model/ItemContainer.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
+package seedu.duke.model;
+
+import java.util.ArrayList;
+
+/**
```
Comment (haoyusimon) [inline src/main/java/seedu/duke/model/ItemContainer.java:5]: Very clear JavaDoc and easy to understand

**Comment by t-l-xin** [comment]
Looks good to merge

**Pull request merged**